src/sidebar/templates/annotation.html

```diff
@@ -58,7 +58,7 @@
        ng-if="(vm.canCollapseBody || vm.state().tags.length) && !vm.editing()">
     <ul class="tag-list" aria-label="Annotation tags">
       <li class="tag-item" ng-repeat="tag in vm.state().tags">
-        <a ng-href="{{vm.tagSearchURL(tag)}}" target="_blank">{{tag}}</a>
+        <a ng-href="{{vm.tagSearchURL(tag)}}" aria-label="Tag {{tag}}" target="_blank">{{tag}}</a>
```

Comment:
Given that tags can contain spaces (I think?!), I think it would be nice to add a colon (`:`) here, e.g.: `aria-label="Tag: {{tag}}"`